Documentation/operations/upgrade.rst

```diff
@@ -288,159 +288,53 @@ communicating via the proxy must reconnect to re-establish connections.
 
 .. _current_release_required_changes:
 
-.. _1.17_upgrade_notes:
+.. _1.18_upgrade_notes:
 
-1.17 Upgrade Notes
+1.18 Upgrade Notes
```

Comment:
1.18 Upgrade Notes section was added on the main branch, so you need to rebase

Reply:
Thanks and I will rebase it once I resolve all the issues